.github/workflows/spelling.yml

```diff
@@ -0,0 +1,18 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Check Website Links"
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - "1.11.x"
+
+jobs:
+  typos:
+    name: "Check for typos"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Check for typos"
+        uses: crate-ci/typos@master
```

Comment:
I don't think this can work. You're missing the "checkout" action. Currently the action is not running at all, you're missing `pull_request` in the `on` section.

Also please pin the action to a specific tag.

Reply:
I am sorry. I am very bad at improvising.

All my workflows are public.
https://github.com/szepeviktor/byte-level-care/tree/master/.github/workflows